src/library/styles/components/_textarea.scss

```diff
@@ -0,0 +1,60 @@
+.bsds-textarea {
+  @extend %field-control;
+  max-width: 100%;
+  min-width: 100%;
+  min-height: space(12x);
+  border-color: $neutral-40;
```

Comment:
can we align with text inputs?

Reply:
should we use $border-color-assertive to align with %field-control and other inputs?